main.go

```diff
@@ -143,6 +152,11 @@ func initMetricsServer(logger *zap.Logger) {
 		return
 	}
 	logger.Info("Starting http metrics in " + metricsAddress)
+	hostname, found := os.LookupEnv("HOSTNAME")
```

Comment:
I think that there is a specific function called `os.Hostname`.

Reply:
Thanks! Updated to use that function